modules/scalaz/src/main/scala/pureconfig/module/scalaz/instances/package.scala

```diff
@@ -0,0 +1,57 @@
+package pureconfig.module.scalaz
+
+import com.typesafe.config.ConfigValue
+import pureconfig.{ ConfigConvert, ConfigReader, ConfigWriter }
+import pureconfig.error.{ ConfigReaderFailure, ConfigReaderFailures, FailureReason }
+
+import scalaz.{ Contravariant, Equal, InvariantFunctor, MonadError, Semigroup, Show }
+
+package object instances {
```

Comment:
Can you add some simple Scaladoc in all the objects? Copying the description from the cats module is fine.

Reply:
Is this enough - c3543de?